src/components/WithdrawRewards/index.tsx

```diff
@@ -105,19 +108,29 @@ function WithdrawRewards({ trigger, type, onClaim, ...rest }: WithdrawRewardsPro
                         </>
                     ) : (
                         <>
-                            <Title className="flex flex-grow justify-center pt-3">Success!</Title>
+                            <Title className="flex flex-grow justify-center pt-3">
+                              {status !== 'success' ? i18n._(t`Success!`) : i18n._(t`Congratulations!`)}
+                            </Title>
                             <div className="flex justify-center items-center gap-2 pt-7 pb-7">
-                                {i18n._(t`Transaction was sent to the blockchain`)}{' '}
-                                {transactionHash && (
+                                { status !== 'success' ?
+                                  i18n._(t`Transaction was sent to the blockchain `) :
+                                  i18n._(t`You have successfully claimed your rewards `)}
```

Comment:
it is not successfully until the TX is confirmed. I think how it was before is fine. plus show the TX hash link

Reply:
that's why the status !== success is there for the first text